lib/services/devices/devices-NGSI-v2.js

```diff
@@ -265,7 +265,7 @@ function createInitialEntityNgsi2(deviceData, newDevice, callback) {
  */
 function updateEntityNgsi2(deviceData, updatedDevice, callback) {
     const options = {
-        url: config.getConfig().contextBroker.url + '/v2/entities/' + String(deviceData.name) + '/attrs',
+        url: config.getConfig().contextBroker.url + '/v2/entities/' + String(deviceData.name) + '/attrs?options=upsert',
```

Comment:
This operation is not going to work... CB only accepts `?options=upsert` in `POST /v2/entities` not in `POST /v2/entities/{id}/attrs`...

In addition, maybe `if (config.getConfig().appendMode === ...)` should be used also here?

Reply:
Then I'll remove `?options=upsert` for that case. That case ins not the usual flow. An check apppendMode is not expected here

Reply:
That flow is for update a register, and has not being modified; this PR just use upsert when update an entity after get a measure